subsys/bluetooth/host/iso.c

```diff
@@ -1504,7 +1506,7 @@ void hci_le_big_sync_established(struct net_buf *buf)
 	BT_DBG("BIG[%u] %p sync established", big->handle, big);
 
 	if (evt->status || evt->num_bis != big->num_bis) {
-		big_disconnect(big);
+		big_disconnect(big, BT_HCI_ERR_UNSPECIFIED);
```

Comment:
If it is evt->status, wouldn't it be best to give that as a reason?

Reply:
Good catch!